package.json

```diff
@@ -52,22 +52,23 @@
     "@rdfjs/data-model": "^1.1.1",
     "@shexjs/core": "^1.0.0-alpha.4",
     "@shexjs/parser": "^1.0.0-alpha.4",
-    "@solid/react": "^1.6.0",
+    "@solid/query-ldflex": "^2.6.0",
+    "@solid/react": "^1.8.0",
     "file-type": "^10.7.1",
     "html-webpack-include-assets-plugin": "^1.0.6",
     "jsonld": "^1.6.2",
     "ldflex": "^2.4.0",
     "lodash": "^4.17.15",
     "mime-types": "^2.1.24",
     "moment": "^2.24.0",
-    "n3": "^1.1.2",
+    "n3": "^1.1.1",
     "rdflib": "^0.20.1",
     "react-color": "^2.17.3",
     "react-datepicker": "^2.9.6",
     "react-router-dom": "^4.3.1",
     "react-select": "^2.2.0",
     "solid-auth-client": "^2.3.0",
-    "solid-forms": "0.0.14-test",
+    "solid-forms": "0.0.17-test",
```

Comment:
In this case, I think we need to regenerate the package-lock as well, or at least make sure it gets updated. I checked the version in the PR and it's still pointing at file:../, even though the package.json has been updated.